### PR TITLE
Tweak style in distribution point handling

### PR DESCRIPTION
**Pull request by djc**
Following up on #127, meant something more like this in my latest review.

### CODE REVIEW

File: src/cert.rs [hunk 1]
```diff
@@ -307,21 +307,18 @@ impl<'a> CrlDistributionPoint<'a> {
             //   a DistributionPoint MUST NOT consist of only the reasons field; either distributionPoint or
             //   cRLIssuer MUST be present.
             match (result.distribution_point, result.crl_issuer) {
-                (None, None) => return Err(Error::MalformedExtensions),
-                _ => {}
+                (None, None) => Err(Error::MalformedExtensions),
```
Comment (djc) [inline src/cert.rs:310]: This brings the two return values closer together, clearly showing that this the final return value.

File: src/cert.rs [hunk 2]
```diff
@@ -307,21 +307,18 @@ impl<'a> CrlDistributionPoint<'a> {
...
     }
 
     /// Return the distribution point names (if any).
     #[allow(dead_code)] // TODO(@cpu): remove this once used in CRL validation.
     pub(crate) fn names(&self) -> Result<Option<DistributionPointName<'a>>, Error> {
-        Ok(match self.distribution_point {
-            None => None,
-            Some(der) => Some(DistributionPointName::from_der(der)?),
-        })
+        self.distribution_point
```
Comment (djc) [inline src/cert.rs:319]: This way, we can avoid the `None => None` arm.

Comment (cpu) [reply src/cert.rs:319]: :sweat_smile: I ended up revisiting this in #128 for the same reason:
https://github.com/rustls/webpki/blob/4601fb6cbcc92fa86c029e435009299e43c10123/src/cert.rs#L317-L323

File: src/cert.rs (hunk 1 above)
Comment (cpu) [reply src/cert.rs:310]: Ahh! I see, thanks. Sorry that I misunderstood the first time :+1:

**Review by cpu — APPROVED** [review]

**Pull request merged**